### DISCUSSION TITLE
The Next auth is working fine on local  it throws [next-auth][error][CLIENT_FETCH_ERROR]

### DISCUSSION THREAD

**Discussion by VyshnavSuresh** [nextauthjs/next-auth#7489] [Help]
### Question 💬

The Next auth is working fine on local 
it throws [next-auth][error][CLIENT_FETCH_ERROR] 

### How to reproduce ☕️

The Next auth is working fine on local and on vercel it throws [next-auth][error][CLIENT_FETCH_ERROR] 

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by seeratawan01** [comment]
I was getting the same error in production and I'm using Prisma with next-auth,

I have fixed it by updating the Build command:
`prisma generate && next build`

And also setting the `NEXTAUTH_SECRET` env

**Reply by zhouyang-bigdata** [reply]
For my project, this was the best match among the 10 answers.

**Comment by diegyn** [comment]
Estou com o mesmo problema em produção na vercel

localhost funciona bem, mas o deploy em produção dá o erro

estou usando CredentialsProvider

e todas as variáveis de ambientes foram definidas
